packages/record/README.md

```diff
@@ -97,3 +99,278 @@ stop or start recording trials within the timeline.
 ```javascript
 jsPsych.run([videoConfig, startRec, morning, evening, stopRec, night]);
 ```
+
+## Video Consent
+
+Users will need to record them accepting the consent document for your study.
```

Comment:
Suggest "Users will need to record **themselves** accepting..."

Reply:
Oh wait, so I thought "users" meant participants here, but maybe it refers to researchers? If so, maybe something like: "Users will need to record their participants accepting..."